tools/purge_orphans.rb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+require "optimist"
+
+VALID_MODES = %w[count purge].freeze
+opts = Optimist.options do
+  banner "Purge orphaned rows from several tables.\n\nUsage: ruby #{$PROGRAM_NAME} [options]\n\nOptions:\n\t"
```

Comment:
There's a better way to do this with optimist I think but this works and I'm not at my computer to try it out

Reply:
thanks, it's `permitted`, pushing that

```ruby
opt :mode, "Mode (#{VALID_MODES.join(", ")})", :default => "count", :permitted => VALID_MODES
```

Reply:
I'm not sure what you mean.

Reply:
You can use synopsis and usage helpers and don't need to reimplement that...For example:

```suggestion
  synopsis "Purge orphaned rows from several tables."
  usage "[options]"
```

Gives you basically the same thing.

Reply:
ah, nice... done